packages/atomic/src/components/insight/atomic-insight-result-attach-to-case/atomic-insight-result-attach-to-case.tsx

```diff
@@ -0,0 +1,127 @@
+import {AttachToCase, buildAttachToCase, Result} from '@coveo/headless/insight';
```

Comment:
Did you consider splitting this into two components (e.g., `atomic-insight-result-attach-to-case-action` and `atomic-insight-result-attach-to-case-indicator`)?

It feels like a somewhat odd design to me to have the same component used in two different ways based on the value of a prop.

Reply:
Good point, I agree it would make more sense. @mikegron what do you think?

Reply:
J'enleverais la prop au complet, ca ne fait "rien" anyway si personne se hook sur l'event donc ont a pas besoin de ca (j'imagine pour le builder?)

Reply:
> It feels like a somewhat odd design to me to have the same component used in two different ways based on the value of a prop.

I agree with this. I think it should be two components.

Reply:
I will make two components. Thanks for your feedback